fastjet/clustersequence.go

```diff
@@ -69,8 +72,63 @@ func NewClusterSequence(jets []Jet, def JetDefinition) (*ClusterSequence, error)
 	return cs, err
 }
 
+//returns the number of exclusive jets that would have been obtained
```

Comment:
the Go style recommends to document functions like so:
```go
// NrExclusiveJets returns the number of ...
func (cs *ClusterSequence) NrExclusiveJets(...) { ...
```
also, `NrExclusiveJets` doesn't look too Go-ish.
see:
```
$> go doc flag.NArg 
func NArg() int
    NArg is the number of arguments remaining after flags have been processed.

$> go doc reflect.Type.NumField
// NumField returns a struct type's field count.
// It panics if the type's Kind is not Struct.
func NumField() int
```

so, well, even though the standard library (itself) doesn't follow a consistent style, `NrFoo` isn't part of that set of inconsistent options :)

I would go with `NumExclusiveJets`.